DashAI/back/plugins/utils.py

```diff
@@ -0,0 +1,44 @@
+from typing import List
+
+
+def _get_all_plugins() -> List[str]:
+    """
+    Make a request to PyPI server to get all package names.
+
+    Returns
+    ----------
+    List[str]
+        A list with the names of all PyPI packages
+    """
+    return [
+        "dashai-tabular-classification-package",
+        "pytorch",
+        "sklearn",
```

Comment:
What is this?
If these are the possible names of the plugins, I don't think it's a good idea that they don't have the dashai prefix.

Reply:
If this is decided, there might even be somewhere some check that the plugins name starts with dashai.

Reply:
This is to show that the function filters out plugins that do not start with the prefix "dashai-".